### ISSUE TITLE
Cant run application anymore

### ISSUE THREAD

**Issue by schamane** [SAP/ui5-webcomponents-react#1260]
Just updated components to:

```json
{
    "@ui5/webcomponents": "^1.0.0-rc.11",
    "@ui5/webcomponents-base": "^0.28.0",
    "@ui5/webcomponents-fiori": "^1.0.0-rc.11",
    "@ui5/webcomponents-icons": "^1.0.0-rc.11",
    "@ui5/webcomponents-react": "0.13.7",
    "@ui5/webcomponents-react-base": "^0.13.7",
    "@ui5/webcomponents-theme-base": "^1.0.0-rc.11",
}
````

and get this error on npm run start

```bash
./node_modules/@ui5/webcomponents-react/lib/ToolbarSeparator.js
Module not found: Can't resolve '@ui5/webcomponents-react-base/hooks/useI18nBundle' in '/Users/user/git/folder/project/web/node_modules/@ui5/webcomponents-react/lib'
```

**Comment by schamane** [comment]
Sorry, please ignore this bug, looks like problem on my local machine

**Comment by schamane** [comment]
No, still dont work.
It works if i downgrade to 0.13.1

**Comment by Lettucez** [comment]
I am also receiving this error when upgrading to to 0.13.7. I was previously on 0.13.4.

**Comment by schamane** [comment]
https://github.com/SAP/ui5-webcomponents-react/commit/7835cc54518b0c39c3d0d250c84f2753868f859e#r46804820

**Comment by MarcusNotheis** [comment]
Thanks for reporting guys - we just released `@ui5/webcomponents-react@0.13.8` containing a fix for that bad import path.

Sorry for the inconvenience!
